hadoop-master1/files/etc/flink/conf/flink-conf.yaml

```diff
@@ -105,16 +105,18 @@ parallelism.default: 1
 # Supported backends are 'hashmap', 'rocksdb', or the
 # <class-name-of-factory>.
 #
-# state.backend.type: hashmap
+state.backend.type: hashmap
+
+state.checkpoints.num-retained: 3
 
 # Directory for checkpoints filesystem, when using any of the default bundled
 # state backends.
 #
-# state.checkpoints.dir: hdfs://namenode-host:port/flink-checkpoints
+state.checkpoints.dir: hdfs://hadoop-master1.orb.local:8020/flink-checkpoints
```

Comment:
will flink auto create the folder if not exist?

Reply:
> will flink auto create the folder if not exist?

Yes, I have verified.